### PR TITLE
S1

### PR DESCRIPTION
**Pull request by objen**
Story 1 - Get all rooms

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Route::controller(RoomController::class)->group(function () {
+    Route::get('/rooms', 'all');
+    Route::get('/rooms/{id}', 'find');
+});
+
+Route::controller(TypeController::class)->group(function () {
+    Route::get('/types', 'all');
+    Route::get('/rooms/{id}', 'find');
+});
```
Comment (ashleycoles) [inline routes/api.php]: Please remove any routes that are not part of this story

File: database/factories/RoomFactory.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'name' => $this->faker->sentence(3),
+            'min_capacity' => 1,
```
Comment (ashleycoles) [inline database/factories/RoomFactory.php]: I think it would be worth brining in a random min as well

File: database/factories/TypeFactory.php [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        return [
+            'name' => $this->faker->sentence(3),
```
Comment (ashleycoles) [inline database/factories/TypeFactory.php:20]: 3 sentences is probs a bit long for a type name really

File: routes/api.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+//Route::controller(TypeController::class)->group(function() {
+//    Route::get('/bookings', 'all');
+//    Route::get('/bookings/{id}', 'find');
+//    Route::get('/bookings/report', 'find');
+//});
```
Comment (ashleycoles) [inline routes/api.php]: Remove commented out code

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: routes/api.php (hunk 1 above)
Comment (eemakim) [reply routes/api.php]: removed :)

File: database/factories/RoomFactory.php (hunk 2 above)
Comment (eemakim) [reply database/factories/RoomFactory.php]: done :)

File: database/factories/TypeFactory.php (hunk 3 above)
Comment (eemakim) [reply database/factories/TypeFactory.php:20]: Doesn't this make a sentence with 3 words in it?

File: routes/api.php (hunk 4 above)
Comment (eemakim) [reply routes/api.php]: Removed :)

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**